networking/synthetic/stac-s2i-builder-image/Dockerfile

```diff
@@ -43,5 +43,5 @@ RUN set -x && config_repo=`echo "${github_url_stac_config}" | rev | cut -d / -f
     && source ${config_repo%.git*}/stac_config \
     && wget -q ${Onload_tar_url} \
     && onload_tar_file=`echo "$Onload_tar_url" | rev | cut -d / -f 1 | rev` \
-    && rpmbuild -tb $onload_tar_file --quiet
-
+    && rpmbuild -tb $onload_tar_file --quiet \
```

Comment:
Eldad, I realize you're following the same pattern as before but can you please move to the conventional pattern for these:
`rpmbuild -tb $onload_tar_file --quiet && \`